src/FamilyHubs.ServiceDirectory.Shared/Display/GeneralDisplayExtensions.cs

```diff
@@ -0,0 +1,11 @@
+﻿
+namespace FamilyHubs.ServiceDirectory.Shared.Display;
+
+public static class GeneralDisplayExtensions
+{
+    //todo: better name?
+    public static string GetDisplay(this string? description)
```

Comment:
thought: maybe `StringOrNone` or similar?

Reply:
Naming is often the hardest part 😄 - we can revisit later.